tests/python/pants_test/engine/legacy/test_changed_integration.py

```diff
@@ -26,6 +26,14 @@ def lines_to_set(str_or_list):
     return set(x for x in str(str_or_list).split('\n') if x)
 
 
+def create_file_in(worktree, path, content):
+  """Creates a file in the given worktree, and returns its path."""
+  write_path = os.path.join(worktree, path)
+  with safe_open(write_path, 'w') as f:
+    f.write(dedent(content))
+  return write_path
```

Comment:
could probably re-use `BaseTest.{create_file,create_workdir_file}` here instead?

Reply:
The workdir in that case is different from the one in use here, unfortunately.